[FR-299] Change image format

hmdros opened the pull request:
Addressing performance improvement suggested by lighthouse reports changed this image to .webp format in internal PR.

hmdros committed:
changing image format

kindoflew commented:
Support for `webp` is pretty good (https://caniuse.com/webp), but I wonder if we should also have `jpeg` as a fallback, just in case? what do you think, @etagwerker?

etagwerker commented:
@kindoflew I like your idea 👍

hmdros committed:
Add fallback with @supports

hmdros commented:
Thanks for the suggestions @kindoflew

I added this @supports CSS rule based on [this blog post](https://www.irigoyen.dev/blog/2021/02/11/image-fallbacks-in-css-using-supports/). Let me know if you have any other opinion.

Also moved back the .jpg images in FR pull request

kindoflew commented on vendor/assets/stylesheets/fastruby/styleguide/4-molecules/_location.scss:
```diff
@@ -2,16 +2,25 @@
   padding: 13rem 1rem 6rem 1rem;
-  background-image: asset_url('bg-location-2.webp');
+  background-image: asset_url("bg-location-2.webp");
+  @supports (background-image: asset_url("bg-location-2.webp")) {
```
i think this logic might be backwards. if the `@supports` is `true`, shouldn't the `.webp` image be nested here?

kindoflew commented on vendor/assets/stylesheets/fastruby/styleguide/4-molecules/_location.scss:
```diff
@@ -2,16 +2,25 @@
     background-position: 50% 53%;
-    background-image: asset_url('bg-location-2.webp');
+    background-image: asset_url("bg-location-2.webp");
+    @supports (background-image: asset_url("bg-location-2.webp")) {
```
same as above.

kindoflew commented on vendor/assets/stylesheets/fastruby/styleguide/4-molecules/_location.scss:
```diff
@@ -2,16 +2,25 @@
   @include respond-above(sm) {
-    background-image: asset_url('bg-location.webp');
+    background-image: asset_url("bg-location.webp");
+    @supports (background-image: asset_url("bg-location.webp")) {
```
same as above.

kindoflew reviewed (changes requested):
Good idea with `@supports`! I think the logic needs to be flipped, but that's it!

hmdros committed:
switch logics from supports back

hmdros commented:
> Good idea with `@supports`! I think the logic needs to be flipped, but that's it!

Done! Thanks @kindoflew
